tools/omnissm/pkg/aws/ec2metadata/document.go

```diff
@@ -33,6 +33,7 @@ type Document struct {
 	InstanceId       string `json:"instanceId"`
 	AccountId        string `json:"accountId"`
 	InstanceType     string `json:"instanceType"`
+	ImageId          string `json:"imageId"`
```

Comment:
i'm assuming this is just for the server side side usage.. in which case its unclear where/what the user of this attribute is.  The whole instance doc needs passing from the client for the signature validation to have worked.

Reply:
The client sends the instance identity doc "raw" as a string, this was the struct being used by the registration handler lambda to unmarshal the document after the signature has been verified against the whole.